openpmd_viewer/openpmd_timeseries/plotter.py

```diff
@@ -303,7 +303,11 @@ def show_field_1d( self, F, info, field_label, current_i, plot_range,
         # Get the x axis
         xaxis = getattr( info, info.axes[0] )
         # Plot the data
-        plt.plot( xaxis, F )
+        if F.dtype in [np.complex256, np.complex128, np.complex64]:
+            plot_data = abs(F) # For complex numbers, plot the absolute value
```

Comment:
Should we print a message?
```suggestion
            print('As the field is complex, its absolute value is plotted')
            plot_data = abs(F) # For complex numbers, plot the absolute value
```

Reply:
I think making this visible is a good idea, but a print might be very verbose in scripted/looped logic.

Could we maybe visible label the absolute, e.g., with `|F|` for complex fields in the plot?

Reply:
Good idea ; I just implemented this in the last commit.